cli/commands/evidence/evidence_test.go

```diff
@@ -52,34 +52,40 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-var sock net.Listener
-var server *grpc.Server
+var (
+	sock                 net.Listener
+	server               *grpc.Server
+	evidenceStoreService *service_evidenceStore.Service
```

Comment:
is it needed to store those services?

Reply:
Why do you think they are stored?

Reply:
I meant stored in the file as a global variable. Before that they were only local or discarded (as `_`)

Reply:
sock and server were also stored as global variables - and don't have to. And sometimes also single services. I wanted to make it look consistent. But I can revert it back (and make other variables, e.g. sock and server, local as well if they can be local variables?)